<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditionToolbar</class>
 <widget class="QWidget" name="EditionToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>562</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <property name="locale">
   <locale language="French" country="France"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="notePage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-119</y>
        <width>216</width>
        <height>399</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Notes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="13" column="0" colspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="17" column="0" colspan="2">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="buttonMi">
         <property name="text">
          <string>Mi</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="buttonFa">
         <property name="text">
          <string>Fa</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonDo">
         <property name="text">
          <string>Do</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="buttonSi">
         <property name="text">
          <string>Si</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="buttonLa">
         <property name="text">
          <string>La</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonRe">
         <property name="text">
          <string>Re</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="buttonPause">
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="buttonLonga">
         <property name="text">
          <string>Longue</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="buttonSol">
         <property name="text">
          <string>Sol</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="buttonHalf">
         <property name="text">
          <string>Blanche</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="button8th">
         <property name="text">
          <string>Croche</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="buttonWhole">
         <property name="text">
          <string>Ronde</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="button32th">
         <property name="text">
          <string>32ème</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="buttonQuarter">
         <property name="text">
          <string>Noire</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="button16th">
         <property name="text">
          <string>Demi-Croche</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QPushButton" name="buttonAucun">
         <property name="text">
          <string>Aucun</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QPushButton" name="buttonFlat">
         <property name="text">
          <string>Bémole</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QPushButton" name="buttonSharp">
         <property name="text">
          <string>Dièse</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QPushButton" name="buttonDot">
         <property name="text">
          <string>Pointée</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QPushButton" name="buttonDoubleSharp">
         <property name="text">
          <string>Double Dièse</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QPushButton" name="buttonDoubleFlat">
         <property name="text">
          <string>Double Bémole</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QPushButton" name="buttonDoubleDot">
         <property name="text">
          <string>Double Pointée</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QPushButton" name="buttonOctavePlus">
         <property name="text">
          <string>Octave +</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QPushButton" name="buttonOctaveMinus">
         <property name="text">
          <string>Octave -</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mesurePage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>216</width>
        <height>469</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Mesure</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="8" column="1">
        <widget class="QComboBox" name="BattementComboBox">
         <property name="maxCount">
          <number>64</number>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Battement</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Rythme</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Chiffrage</string>
         </property>
        </widget>
       </item>
       <item row="20" column="0" colspan="2">
        <widget class="Line" name="line_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="ClefUt">
         <property name="text">
          <string>Ut</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="ClefSol">
         <property name="text">
          <string>Sol</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="21" column="0">
        <widget class="QPushButton" name="buttonAddMesure">
         <property name="text">
          <string>Ajouter portée</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="ClefFa">
         <property name="text">
          <string>Fa</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QPushButton" name="buttonDieseDo">
         <property name="text">
          <string>Do</string>
         </property>
        </widget>
       </item>
       <item row="21" column="1">
        <widget class="QPushButton" name="buttonDeleteMesure">
         <property name="text">
          <string>Supprimer portée</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QPushButton" name="buttonDieseLa">
         <property name="text">
          <string>La</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QPushButton" name="buttonDieseRe">
         <property name="text">
          <string>Ré</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QPushButton" name="buttonDieseSol">
         <property name="text">
          <string>Sol</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QPushButton" name="buttonDieseFa">
         <property name="text">
          <string>Fa#</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Clefs</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1">
        <widget class="QPushButton" name="buttonDieseSi">
         <property name="text">
          <string>Si</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QPushButton" name="buttonDieseMi">
         <property name="text">
          <string>Mi</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QPushButton" name="buttonBemoleRe">
         <property name="text">
          <string>Ré mineur</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QPushButton" name="buttonDieseDoDiese">
         <property name="text">
          <string>Do#</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Armure</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QPushButton" name="buttonBemoleDo">
         <property name="text">
          <string>Do mineur</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QPushButton" name="buttonBemoleSi">
         <property name="text">
          <string>Si bémole mineur</string>
         </property>
        </widget>
       </item>
       <item row="16" column="1">
        <widget class="QPushButton" name="buttonBemoleSol">
         <property name="text">
          <string>Sol mineur</string>
         </property>
        </widget>
       </item>
       <item row="17" column="1">
        <widget class="QPushButton" name="buttonBemoleFa">
         <property name="text">
          <string>Fa mineur</string>
         </property>
        </widget>
       </item>
       <item row="19" column="0">
        <widget class="QPushButton" name="buttonBemoleLa">
         <property name="text">
          <string>La bémole mineur</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="QPushButton" name="buttonBemoleMi">
         <property name="text">
          <string>Mi bémole mineur</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSlider" name="RythmeSlider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>64</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="informationPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>230</width>
        <height>280</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Informations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Compositeur</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Titre</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="titre"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="compositeur"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="previewnote" native="true">
     <property name="minimumSize">
      <size>
       <width>110</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonAjouterNote">
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
